### PR TITLE
backend/handlers: split chart data into daily and hourly data sets

### PR DESCRIPTION
**Pull request by benma**
Daily: all time
Hourly: only last week

### CODE REVIEW

File: frontends/web/src/routes/account/summary/accountssummary.tsx [hunk 1]
```diff
@@ -53,7 +53,8 @@ interface Response {
     totals: Totals;
     coinNames: CoinNames;
     chartDataMissing: boolean;
-    chartData: ChartData;
+    chartDataDaily: ChartData;
+    chartDataHourly: ChartData;
```
Comment (x1ddos) [inline frontends/web/src/routes/account/summary/accountssummary.tsx:57]: This doesn't seem to be used at all. What is it for?

Comment (x1ddos) [reply frontends/web/src/routes/account/summary/accountssummary.tsx:57]: I guess it's for later use?

**Review by x1ddos — APPROVED** [review]
LGTM

File: frontends/web/src/routes/account/summary/accountssummary.tsx (hunk 1 above)
Comment (benma) [reply frontends/web/src/routes/account/summary/accountssummary.tsx:57]: yeah @thisconnect will use it with the timeframe buttons.

**Pull request merged**